public/index.html

```diff
@@ -1,9 +1,27 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <!-- HTML Meta Tags -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <title>Micrometer Application Observability</title>
+    <meta name="description" content="undefined">
+
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://micrometer.io/">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Micrometer Application Observability">
+    <meta property="og:description" content="undefined">
+    <meta property="og:image" content="">
```

Comment:
Did you generate these via https://www.opengraph.xyz?
If so, it might worth a word in the description+commit message.

Do you happen to know why the `description` is `undefined` and the `image` is empty?